meson.build

```diff
@@ -200,6 +200,15 @@ else
     summary({'execinfo' : ['stacktrace supported:', false]}, section : 'Debugging', bool_yn : true)
 endif
 
+# TODO(xsdg): Also force-enable unit tests in developer mode.
```

Comment:
This comment is an aside to the PR. 
The Ddevel= option is badly named. It should be -Dextended-stacktrace= and should be enabled by default. If the packagers do not want it, that is up to them to disable it. But I think that if a user experiences a crash, the developers will want it. Attached is an example of the output.
I also used that option as a flag for something else - usually a bad idea. If someone clones the repo. for a local build, it is not necessary for the test images to be download. That should be another option.
[stacktrace.txt.gz](https://github.com/BestImageViewer/geeqie/files/15282585/stacktrace.txt.gz)

Reply:
Got it; dropped this TODO from the original commit.

That said, this is clearly super helpful, but I'd wonder how much it might affect binary size, and whether it has an impact on execution speed.